pkglistgen.py

```diff
@@ -770,6 +770,16 @@ def setup_tool(self):
             tool.architectures = ARCHITECTURES
         return tool
 
+    def _product_converter_enabled(self, api, project, product):
+        url = api.makeurl(['source', project, product, '_service'])
+        servicefile = ET.parse(http_GET(url)).getroot()
+
+        for service in servicefile.findall('./service[@name="product_converter"]'):
+            if 'mode' in service.attrib and service.attrib["mode"] == "disabled":
```

Comment:
I think you can write this easier as 
```
return service.find('./service[@name="product_converter" and @mode="disabled"]') is None

Reply:
just changed it to `return not servicefile.findall('./service[@name="product_converter"][@mode="disabled"]')`